synapse/cortex.py

```diff
@@ -3348,6 +3350,103 @@ def _loadCoreModule(self, ctor, conf=None):
         '''
         return await self.view.listTriggers()
 
+    @staticmethod
+    def _convert_reqdict(reqdict):
+        return {s_agenda.TimeUnit.fromString(k): v for (k, v) in reqdict.items()}
+
+    async def addCronJob(self, user, query, reqs, incunit=None, incval=1):
```

Comment:
With these new methods, CoreAPI already implements many of them. Can you make the coreapi call the Cell implementation so we're not repeating ourselves ?  Future selves will thank us.

Reply:
* Can you make the coreapi call the Cell implementation * with the prerequisite permission checks of course :)

Reply:
Also, is it intended that we will remove the CoreApi imlementations in 0.3.0 ? if so those would need deprecation warnings as well.

@jnwatson

Reply:
CoreAPI updated to call the Cell implementations.